recipe/meta.yaml

```
@@ -36,11 +40,26 @@ requirements:
     - libpqxx =6.4
     - spdlog
 
-test:
-  commands:
-    - test -f ${PREFIX}/lib/libhdb++timescale${SHLIB_EXT}
-    - test -f ${PREFIX}/lib/cmake/libhdbpp-timescale/libhdbpp-timescaleConfig.cmake
+outputs:
+  - name: "libhdbpp-timescale"
+    files:
+      - "lib/libhdb++timescale.so"
+      - "lib/libhdb++timescale.so.2"
+      - "lib/libhdb++timescale.so.{{ version }}"
```

All filed have to be listed. We have to specify the cmake files. Specifying the dir should be fine:

lib/cmake/libhdbpp-timescale/

---

Not sure what you mean here; should the section be
```yaml
   files:
     - lib/cmake/libhdbpp-timescale/
```

---

Just add that path to the existing section:

```suggestion
      - "lib/libhdb++timescale.so.{{ version }}"
      - lib/cmake/libhdbpp-timescale/
```